sigopt_sklearn/search.py

```diff
@@ -274,7 +276,8 @@ def _fit(self, X, y, parameter_iterable=None):
         pre_dispatch = self.pre_dispatch
 
         # setup SigOpt experiment and run optimization
-        self._create_sigopt_exp(self.sigopt_connection)
+        if self.experiment is None:
+            self.experiment = self._create_sigopt_exp(self.sigopt_connection)
```

Comment:
I like it, as long as we still have the fully automatic workflow.  I think exposing the create_experiment method as public is fine.